signer/tests/integration/postgres.rs

```diff
@@ -129,7 +133,6 @@ async fn should_be_able_to_query_bitcoin_blocks() {
             .expect("failed_to_execute_query");
         assert!(result.is_none());
     }
-    signer::testing::storage::drop_db(store).await;
```

Comment:
This call got stuck for some reason, maybe I was forgetting some open connection somewhere.

Reply:
Does the database automatically disappear when the test ends only if the test succeeds?

Reply:
If a test concludes "naturally" (succeeding or failing, even with panics) the stack is downed. If you cancel a test (eg,  SIGINT) then the stack is not downed and will be still up after the run finishes.

About the database: the compose client thingy we use by default uses `--volumes` when `docker compose down`, so even if we were to use volumes for the db it should disappear when the stack is downed; but given we are not even using volumes for the test stack, it should just disappear once the stack is removed.